pkg/services/cosmosdb/mongodb-types.go

```diff
@@ -8,4 +8,5 @@ type mongoCredentials struct {
 	Username         string `json:"username,omitempty"`
 	Password         string `json:"password,omitempty"`
 	ConnectionString string `json:"connectionString,omitempty"`
+	URI              string `json:"uri,omitempty"`
```

Comment:
Let's drop `omitempty` here. We should also drop it on the lines above. I know why it was there...

Back before you split this into multiple service managers, there was a single `GetCredentials(....)` function that had to return credentials of a few different "shapes." That isn't a concern anymore and we can reduce cognitive load on contributors by leaving off `omitempty` on field we know will always be set to non-zero values.

Reply:
I went ahead and cleaned all of these up, both on the mongo and the cosmos generic one. With the specific types, all the things will always be set.